lib/TabTracker.js

```diff
@@ -0,0 +1,89 @@
+const tabs = require("sdk/tabs");
+const ss = require("sdk/simple-storage");
+
+const NEWTAB_URL = "resource://activity-streams/data/content/activity-streams.html#/";
+
+const TabTracker = {
+  _openTabs: [],
+
+  init: function() {
+    this.onOpen = this.onOpen.bind(this);
```

Comment:
I have to look at what `this` is bound to, but this is used elsewhere in the code. Is that even necessary?